app/models/question.rb

```diff
@@ -1,7 +1,10 @@
 class Question < ActiveRecord::Base
+  has_many :choices
   validates :body, presence: { message: "Question can't be blank" }
   validates :body, length: {
     maximum: 140,
     message: "Question must be less than 140 characters"
   }
+  accepts_nested_attributes_for :choices, reject_if: proc { |attributes| attributes["description"].blank?
```

Comment:
Line is too long. [105/80]<br>Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.